### PR TITLE
Add back in function used by touchforms

### PR DESCRIPTION
**Pull request by wpride**
Add back in a function that was removed here https://github.com/dimagi/javarosa/commit/6aedc0570c521a272b7ca0846764bcd2c52c9dba

Needed by touchforms

@phillipm

### CODE REVIEW

File: core/src/main/java/org/javarosa/core/model/FormIndex.java [hunk 1]
```diff
@@ -438,4 +438,28 @@ public static boolean overlappingLocalIndexesMatch(FormIndex parent, FormIndex c
         }
         return parent.getLocalIndex() == child.getLocalIndex();
     }
+
+    public void assignRefs(FormDef f) {
```
Comment (phillipm) [inline core/src/main/java/org/javarosa/core/model/FormIndex.java]: Can you mention that this is needed by touchforms so I don't remove it in the future?

File: core/src/main/java/org/javarosa/core/model/FormDef.java [hunk 2]
```diff
@@ -1365,6 +1365,8 @@ public void populateDynamicChoices(ItemsetBinding itemset, TreeReference curQRef
 
         }
 
+        System.out.println("121 populate choices: " + choices);
```
Comment (phillipm) [inline core/src/main/java/org/javarosa/core/model/FormDef.java]: Would be nice to not include this debug print

**Comment by phillipm** [comment]
For touchforms jar deployment, do you want this in 2.26 or is it fine just being on master?

**Comment by wpride** [comment]
doesn't particularly matter

**Pull request merged**